Source/OxyPlot/Series/FinancialSeries/VolumeSeries.cs

```diff
@@ -423,6 +423,11 @@ protected internal override void UpdateData()
             base.UpdateData();
             this.winIndex = 0;
 
+            if (this.data == null || this.data.Count == 0)
```

Comment:
Are we changing behaviour here in case `data != null` and `data.Count == 0`? Previously it that case line 447 would have been run...

Reply:
Yes, the UpdateData behavior is changed for this case, but it will have no effect on the plot since minDx is a gap size between existing bars. But, we have no bars at all if data.Count is 0, so minDx is not used.